src/zmq/zmqpublishnotifier.cpp

```diff
@@ -2,14 +2,25 @@
 // Distributed under the MIT software license, see the accompanying
 // file COPYING or http://www.opensource.org/licenses/mit-license.php.
 
+#if defined(HAVE_CONFIG_H)
+#include <config/bitcoin-config.h>
+#endif
+
+#include <zmq/zmqpublishnotifier.h>
+
 #include <chain.h>
 #include <chainparams.h>
 #include <streams.h>
-#include <zmq/zmqpublishnotifier.h>
+#include <zmq/zmqutil.h>
 #include <validation.h>
 #include <util/system.h>
 #include <rpc/server.h>
 
+#include <zmq.h>
+
+#include <cstdarg>
+#include <cstddef>
```

Comment:
```suggestion
#include <cstdarg>
#include <cstddef>
#include <map>
#include <string>
#include <utility>
```

Reply:
Indeed, I've pushed this change together with the two nanonits below (in a single commit).